src/kirby/components/empty-state/empty-state.component.html

```diff
@@ -0,0 +1,8 @@
+<article>
+  <kirby-icon *ngIf="iconName" [name]="iconName" size="lg" outline="true"></kirby-icon>
+  <h3 *ngIf="title">{{ title }}</h3>
+  <p *ngIf="text">{{ text }}</p>
+  <div class="button-wrapper">
```

Comment:
Do you need to wrap the content?
```suggestion
  <ng-content></ng-content>
```

Reply:
This is needed for us to be able to apply a margin-top to the content.